client/src/pages/Register.js

```diff
@@ -75,6 +75,7 @@ const Register = ({ enqueueSnackbar }) => {
       .then(res => {
         if (res.data.success) {
           localStorage.setItem("email", res.data.user.username);
+          localStorage.setItem("isVerified", res.data.user.isVerified);
```

Comment:
Wouldn't this always be false?

Reply:
It will be false right after the register, and it becomes true once they click on the dashboard.

Reply:
Got it, I just meant that you don't necessarily need to set it because localStorage.getItem(..) returns false if the key isn't there